schemas/expconf/v0/log-pattern-policy.json

```diff
@@ -0,0 +1,25 @@
+{
+    "$schema": "http://json-schema.org/draft-07/schema#",
+    "$id": "http://determined.ai/schemas/expconf/v0/log-pattern-policy.json",
+    "title": "LogPatternPolicy",
+    "additionalProperties": false,
+    "required": [
+        "pattern"
+    ],
+    "type": "object",
+    "properties": {
+        "pattern": {
+            "type": [
+                "string"
+            ]
+        },
+        "policy": {
+            "type": [
+                "object",
```

Comment:
Originally policy was a union since some of the types took extra keys. Now this isn't the case but I still like think it makes sense to keep it nested to future proof needing additional parameters per different policy type.

I don't think it matters too much when there isn't additional parameters but when we potentially add another policy I like allowing a separation between what triggers the policy and the policy itself.

Reply:
That's a good explanation. I buy it. Thanks!

Reply:
Ryan's feedback here is to prefer actions be union of types even if, for now, all types take the same arguments. Error messaging is better for unions.

Do you have a sense of how difficult it would be to turn the enum into an object at a later date, should that be something we want to do? From just a front-end, syntax point of view, it doesn't seem like breaking change.

Reply:
Actions into a union